src/main/resources/mediathek/res/programm/fxml/film_toolbar.fxml

```diff
@@ -18,7 +18,7 @@
         </graphic>
     </Button>
 
-    <VerticalSeparator/>
```

Comment:
as well, what is the benefit?

Reply:
This allows using the "build-in" separator definition of JavaFX instead of creating a new object (mediathek.javafx.VerticalSeperator) which does the same and adds no benefit.